doc/articles/how-uno-works.md

```diff
@@ -2,60 +2,110 @@
+The rest of this article explains how Uno enables WinUI-compatible XAML and C# apps to run on **non-Windows platforms** using `Uno.UI`.
 
-## Uno.UI at runtime
+## Uno.UI at Runtime
 
-The [`Uno.WinUI` NuGet package](https://www.nuget.org/packages/Uno.WinUI/) completely reproduces the WinUI API surface: all namespaces (`Microsoft.UI.Xaml`, `Windows.Foundation`, `Windows.Storage`, etc), all classes, all class members. Insofar as possible, the same look and behavior as on Windows are replicated on all other platforms.
+The [`Uno.WinUI` NuGet package](https://www.nuget.org/packages/Uno.WinUI/) re-implements the entire WinUI API surface across platforms. This includes:
 
-Note that, as the API surface is very large, some parts are included but not implemented. These features are marked with the `Uno.NotImplementedAttribute` attribute, and a code analyzer is included with the Uno.WinUI package will generate a warning for any such features that are referenced. You can see a complete list of supported APIs [here](implemented-views.md).
+- Namespaces like `Microsoft.UI.Xaml`, `Windows.Foundation`, `Windows.Storage`, and more.
+- Classes, enums, and all relevant members.
 
-## How the UI is rendered
+> [!NOTE]
+> As the API surface is very large, some parts are included but not implemented. These features are marked with the [`Uno.NotImplementedAttribute`] attribute, and a code analyzer is included with the Uno.WinUI package will generate a warning for any such features that are referenced. You can see a complete list of supported APIs [here](implemented-views.md).
 
-As an application developer, you normally don't need to worry about exactly how Uno renders your visual tree to the screen on each platform. For every platform, the public API - panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](xref:Uno.Development.NativeViews) on a platform-specific basis.
+## Rendering the UI
 
-### Web (WebAssembly)
+As a developer, you write your UI using the standard WinUI-based XAML. Under the hood, Uno Platform maps this visual tree to the appropriate rendering mechanism for each platform.
 
-On the web, each XAML element is converted into an appropriate [HTML element](https://developer.mozilla.org/en-US/docs/Glossary/Element). Panels, controls, and other 'intermediate' elements in the visual tree are converted to `<div/>` elements, whereas 'leaf' elements like `TextBlock`, `Image` etc get converted into more specific tags (`<p/>`, `<img/>` etc).
+For every platform, the public API - panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](xref:Uno.Development.NativeViews) on a platform-specific basis.
 
-### iOS and Mac Catalyst
+### Native Rendering Targets
```

Comment:
I would put the `Skia Rendering` section above `Native Rendering` section, as it is the default